lib/elixir/lib/kernel.ex

```diff
@@ -5187,6 +5187,10 @@ defmodule Kernel do
   An overridable function is lazily defined, allowing a developer to override
   it.
```

Comment:
```suggestion
  If the user defines a new function or macro with the same name
  and arity, then the overridable ones are discarded. Otherwise, the
  original definitions are used.
```